_episodes/04-sharing-environments.md

```diff
@@ -349,63 +354,77 @@ The `--prune` option tells Conda to remove any dependencies that are no longer r
 > {: .solution}
 {: .challenge}
 
-> ## Installing via `pip` in `environment.yml` files
->
-> Since you write `environment.yml` files for all of your projects, you might be wondering how
-> to specify that packages should be installed using `pip` in the `environment.yml` file.  Here
-> is an example `environment.yml` file that uses `pip` to install the `kaggle` and `yellowbrick`
-> packages.
->
-> ~~~
-> name: example
+## Installing via `pip` in `environment.yml` files
+
+Since you write `environment.yml` files for all of your projects, you might be wondering how to specify that packages
+should be installed using `pip` in the `environment.yml` file.  Here is an example `environment.yml` file that uses
+`pip` to install the `kaggle` and `yellowbrick` packages.
+
+~~~
+name: example
+
+dependencies:
+  - jupyterlab=1.0
+  - matplotlib=3.1
+  - pandas=0.24
+  - scikit-learn=0.21
+  - pip=22.3
+  - pip:
+    - kaggle
+    - yellowbrick==1.5
+~~~
+
+> Note two things...
 >
-> dependencies:
->  - jupyterlab=1.0
->  - matplotlib=3.1
->  - pandas=0.24
->  - scikit-learn=0.21
->  - pip=22.3
->  - pip:
->    - kaggle==1.5
->    - yellowbrick==1.5
-> ~~~
+> 1. `pip` is installed as a dependency under conda **first** (under the `dependencies` section) with an explicit
+> version (not essential).
+> 2. Following this there is then an entry for `- pip:` and under this is another list (indented further) where double
+> '==' instead of '=' for the explicit version that `pip` will install.
 >
-> Note the double '==' instead of '=' for the `pip` installation and that you should include `pip` itself
-> as a Conda dependency and then a subsection denoting those packages to be installed via `pip`. In case
-> you are wondering, the [Yellowbrick](https://www.scikit-yb.org/en/latest/) package is a suite of
-> visual diagnostic tools called “Visualizers” that extend the
-> [Scikit-Learn](https://scikit-learn.org/stable/) API to allow human steering of the model selection
-> process. Recent version of Yellowbrick can also be installed using `conda` from the `conda-forge` channel.
+> In case you are wondering, the [Yellowbrick](https://www.scikit-yb.org/en/latest/) package is a suite of visual
+> diagnostic tools called “Visualizers” that extend the [Scikit-Learn](https://scikit-learn.org/stable/) API to allow
+> human steering of the model selection process. Recent version of Yellowbrick can also be installed using `conda` from
+> the `conda-forge` channel.
 >
 > ~~~
 > $ conda install --channel conda-forge yellowbrick=1.2 --name project-env
 > ~~~
->
+{: .callout}
+
+

```

Comment:
I think we could lose or move the stuff about requirements.txt.

Reply:
Agreed, excised.